indicators/core/nuts_utils.py

```diff
@@ -92,15 +92,21 @@ def make_reverse_lookup(data, key=itemgetter(1), prefix=""):
 @lru_cache()
 def get_geo_lookup(module):
     # Forward lookup
-    nf = NutsFinder()
-    id_to_nuts_lookup = {
-        id: nf.find(lat=lat, lon=lon) for id, lat, lon in module.get_lat_lon()
-    }
-    id_nuts = [  # splatten out the nuts IDs, ready for grouping
-        (id, info["NUTS_ID"])
-        for id, nuts_info in id_to_nuts_lookup.items()
-        for info in nuts_info
-    ]
+    try:
```

Comment:
note that for Cordis we don't have implement `lat`, `lon` level info, so `cordis_topics` doesn't have a `get_lat_lon` method. Instead it has `get_nuts_to_id` which is triggered by the `AttributeError`